### ISSUE TITLE
Can't set Content-Type header for a GET request

### ISSUE THREAD

**Issue by mjbnz** [Waboodoo/HTTP-Shortcuts#159]
Using netcat listening, I get this for a GET, with `Content-Type: application/json` set:

```
$ nc -l -p 8787
GET / HTTP/1.1
Connection: close
User-Agent: HttpShortcuts/1.25.0 (Linux; U; Android 9; Nokia 7 plus Build/PPR1.180610.011)
Host: <ip>:8787
Accept-Encoding: gzip

```

**Comment by tobyink** [comment]
Although it's technically legal, it's very rare to include a body with an HTTP GET request, so a `Content-Type` header would be meaningless. Did you intend to include an `Accept` header instead?

**Comment by mjbnz** [comment]
You are correct, it's not common... and Technically legal is the best kind of legal (....? ;) ) I did not intend to use an `Accept` header.

Specifically what I was trying was to request a JSON response from an Amazon Dash Button. Yes, it should use the `Accept` header, but for whatever reason, it expects `Content-Type`.

**Comment by Waboodoo** [comment]
Thank you for reporting this. With the current implementation (v1.25) it is not possible to set a `Content-Type` header for a GET request. I will fix this for version 1.26

**Comment by Waboodoo** [comment]
This was fixed with the 1.26 release
